librad/src/net/protocol/io/recv/git.rs

```diff
@@ -110,7 +112,30 @@ where
             .map(|ReplicateResult { updated_tips, .. }| updated_tips);
             match updated_tips {
                 None => tracing::info!("rere skipped"),
-                Some(xs) => tracing::info!("rere updated {} refs", xs.len()),
+                Some(xs) => {
+                    tracing::info!("rere updated {} refs", xs.len());
```

Comment:
Could we log the actual tips here?

Reply:
Hm, really? At what log level? There could be a lot.

Reply:
Yeah maybe at trace, mainly because this kind of information is really useful in ruling things out when debugging why things aren't replicating.

Reply:
How about instrumenting `control::gossip`, so it catches all broadcast messages, and the span path shows where it originated from?

Reply:
Yeah that would serve the same purpose I think. As long as we don't think it's likely we'll add additional conditionals before the gossip.

Reply:
Turns out that doesn't yield much useful output due to `Debug` being too, ehm, inspecting. Went with your suggestion instead.